zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/usermanagement/dto/AssignUserToGroupRequest.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.gateway.rest.controller.usermanagement.dto;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+
+public record AssignUserToGroupRequest(
+    @NotBlank(message = "userId.invalid") @Size(max = 255, message = "userId.invalid") Long userId,
+    String organizationId) {}
```

Comment:
Do we need `organizationId` ?

Reply:
C/p'ed that from old identity ... removed it here now 😉